app/models/node.rb

```diff
@@ -176,6 +176,11 @@ def setup
     save
   end
 
+  def match_preview_tag(key, preview_tags)
+    preview_tag = preview_tags.join(" ").match(key)
```

Comment:
Could this method in general be replaced with `preview_tags.includes?(key)`? (i think that's the syntax) -- without a separate method? Just looking - this code looks fine, but that might be simpler? Thanks!